mffpy/header_block/header_block.py

```diff
@@ -38,64 +38,35 @@
 "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 ANY KIND, either express or implied.
 """
-import struct
-from os import SEEK_CUR
-from typing import IO, Union, Optional, Tuple
+from typing import Optional, Tuple
 from collections import namedtuple
-from io import FileIO
 
 import numpy as np
 
-FileLike = Union[IO[bytes], FileIO]
+from .helpers import FileLike, read, skip, write
+from . import optional_header_block as opt
 
-
-# Magic padding from '/examples/example_1.mff/signal1.bin', so that we survive
-# the tests.  Ask Robert about this!!!
-PADDING = np.array([
-    24, 0, 0, 0,
-    1, 0, 0, 0,
-    189, 0, 0, 0,
-    0, 0, 0, 0,
-    196, 63, 9, 0,
-    0, 0, 0, 0,
-    1, 1, 0, 0],
-    dtype=np.uint8).tobytes()
+HEADER_BLOCK_PRESENT = 1
 
 _HeaderBlock = namedtuple('_HeaderBlock', [
     'header_size',
     'block_size',
     'num_channels',
     'num_samples',
-    'sampling_rate'
+    'sampling_rate',
+    'optional'
```

Comment:
I think we could use a better name for `optional` since it doesn't seem to describe what the field is by itself. I think `optional_header` might be a better alternative.